lib/PuppeteerSharp/Browser.cs

```diff
@@ -132,7 +142,8 @@ private void ChangeTargetInfo(MessageEventArgs args)
                 throw new InvalidTargetException("Target should exists before ChangeTargetInfo");
             }
 
-            var target = _targets[args.MessageData.targetInfo.targetId.Value];
+            string targetId = args.MessageData.targetInfo.targetId.Value;
+            var target = _targets[targetId];
```

Comment:
this way the IDE knows that target is of type `Target` and show that the method `TargetInfoChanged` has a reference 😏

Reply:
And if we do ` _targets[args.MessageData.targetInfo.targetId.Value.ToString()]` ?

Reply:
since the `dynamic` keyword is part of the expression the result is also a dynamic object. The `ToString` would only be evaluated at runtime...